lib/src/Pos/Block/Worker.hs

```diff
@@ -81,7 +82,7 @@ blkWorkers keepAliveTimer =
 
 informerWorker :: BlockWorkMode ctx m => (WorkerSpec m, OutSpecs)
 informerWorker =
-    onNewSlotWorker True announceBlockOuts $ \slotId _ ->
+    onNewSlotWorker True mempty $ \slotId _ ->
```

Comment:
The whole idea of having the workers give these `OutSpecs` was madness to begin with. It implies a tight coupling with the networking/diffusion system. With this change, the diffusion layer explicitly states the `InSpecs`/`OutSpecs`, because it knows them up-front (and they were indeed always known up-front even without this patch). We don't put them on workers anymore.

I suppose we should make a youtrack task to remove `OutSpecs` from workers.

Reply:
> The whole idea of having the workers give these OutSpecs was madness to begin with.

Yes, I totally agree. I just want to make sure that nothing is broken by this change.

> I suppose we should make a youtrack task to remove OutSpecs from workers.

It would be good.